docs/src/css/custom.css

```
@@ -2890,6 +2917,91 @@ nav.theme-doc-breadcrumbs {
   }
 }
 
+/* B2C identity journey — simple clickable node row */
+.uc-b2c-roadmap {
+  --uc-node-w: 9rem;
+  --uc-node-h: 5rem;
+  --uc-icon-size: 4rem;
+  display: flex;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)